tests/integration/net/state/simple/peer/subscribe/with_add_test.go

```diff
@@ -224,103 +228,6 @@ func TestP2PSubscribeAddSingleErroneousCollectionID(t *testing.T) {
 	testUtils.ExecuteTestCase(t, test)
 }
 
-func TestP2PSubscribeAddValidAndErroneousCollectionID(t *testing.T) {
-	test := testUtils.TestCase{
-		Actions: []any{
-			testUtils.RandomNetworkingConfig(),
-			testUtils.RandomNetworkingConfig(),
-			testUtils.SchemaUpdate{
-				Schema: `
-					type Users {
-						name: String
-					}
-				`,
-			},
-			testUtils.ConnectPeers{
-				SourceNodeID: 1,
-				TargetNodeID: 0,
-			},
-			testUtils.SubscribeToCollection{
-				NodeID:        1,
-				CollectionIDs: []int{0, testUtils.NonExistentCollectionID},
-				ExpectedError: "datastore: key not found",
-			},
-			testUtils.CreateDoc{
-				NodeID: immutable.Some(0),
-				Doc: `{
-					"name": "John"
-				}`,
-			},
-			testUtils.WaitForSync{},
-			testUtils.Request{
-				// Nothing should sync, although the collection 0 was valid, as it was included in the same
-				// `Add` call it should have been rolled back.
-				NodeID: immutable.Some(1),
-				Request: `query {
-					Users {
-						name
-					}
-				}`,
-				Results: []map[string]any{},
-			},
-		},
-	}
-
-	testUtils.ExecuteTestCase(t, test)
-}
-
-func TestP2PSubscribeAddValidThenErroneousCollectionID(t *testing.T) {
```

Comment:
question: Why have these tests been removed?

Reply:
`TestP2PSubscribeAddSingleErroneousCollectionID` tests the same thing since only one subscription at a time is allowed.

Reply:
`TestP2PSubscribeAddSingleErroneousCollectionID` does not test that other existing collection-peer subscriptions are unaffected by the attempt to configure a bad collection ID.

Reply:
My bad, I had misunderstood these tests. I restored them with one minor modification to make it pass.